test/box/tx_man.test.lua

```diff
@@ -1182,6 +1182,29 @@ box.execute([[DROP TABLE k3;]])
 box.execute([[DROP TABLE k2;]])
 box.execute([[DROP TABLE k1;]])
 
+-- https://github.com/tarantool/tarantool/issues/6396
+
+assert(not box.txn_id())
```

Comment:
This looks almost as cases in engine/transaction, please leave only tests only in one file - too many tests for such simple thing:)